app/src/main/java/fr/free/nrw/commons/utils/LengthUtils.java

```diff
@@ -37,25 +37,57 @@ public static double computeDistanceBetween(LatLng from, LatLng to) {
         return computeAngleBetween(from, to) * 6371009.0D; // Earth's radius in meter
     }
 
+    /**
+     * Computes angle between two points
+     *
+     * @param from Point A
+     * @param to   Point B
+     * @return Angle in radius
+     */
     private static double computeAngleBetween(LatLng from, LatLng to) {
         return distanceRadians(Math.toRadians(from.getLatitude()),
                 Math.toRadians(from.getLongitude()),
                 Math.toRadians(to.getLatitude()),
                 Math.toRadians(to.getLongitude()));
     }
 
+    /**
+     * Computes arc length between 2 points
+     * @param lat1 Latitude of point A
+     * @param lng1 Longitude of point A
+     * @param lat2 Latitude of point B
+     * @param lng2 Longitude of point B
+     * @return Arc length between the points
+     */
     private static double distanceRadians(double lat1, double lng1, double lat2, double lng2) {
         return arcHav(havDistance(lat1, lat2, lng1 - lng2));
     }
 
+    /**
+     * Gets twice the amount of arc sine of (x)^(1/2)
```

Comment:
"Calculate the inverse haversine of an angle. See https://en.wikipedia.org/wiki/Haversine_formula"